vendor/extensions/events/db/migrate/1_create_events_events.rb

```diff
@@ -12,6 +12,7 @@ def up
       t.string :zipcode
       t.string :registration_link
       t.integer :position
+      t.text :description
```

Comment:
In general it is not advisable to update migrations in a new pull request. Could you make a new migration for this change?